Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM golang:1.11.2 as build-env
+FROM golang:1.14.2 as build-env
```

Comment:
Not sure why but for me docker image won't build with golang `1.11.2` all of a sudden. Bumping to `1.14.2` worked. I can revert it back to `1.11.2` if this is unnecessary.